runtime/Cargo.toml

```diff
@@ -13,39 +13,39 @@ repository = "https://github.com/substrate-developer-hub/substrate-node-template
 targets = ["x86_64-unknown-linux-gnu"]
 
 [dependencies]
-codec = { package = "parity-scale-codec", version = "3.6.1", default-features = false, features = ["derive"] }
-scale-info = { default-features = false, features = ["derive"] }
+codec = { package = "parity-scale-codec", version = "3.6", default-features = false, features = ["derive"] }
+scale-info = { version = "2.5", default-features = false, features = ["derive"] }
```

Comment:
They updated `scale-info` codec to 2.10.0 in the runtime code:
https://github.com/paritytech/polkadot-sdk/blob/fcfdb98ab5c5fce91cce339c8d3e4f5f5844bbe1/substrate/bin/node-template/runtime/Cargo.toml#L17

Reply:
Done